### ISSUE TITLE
Is there a way to change the search engine?

### ISSUE THREAD

**Issue by akkartik** [atlas-engineer/next#145]
Thank you.

**Comment by jmercouris** [comment]
Yes, it would involve changing and overriding the URL handling in your init file, there is no simple way to do it, but there is a way to do it. Please take a look at PARSE-URL and GENERATE-SEARCH-QUERY within utility.lisp.

**Comment by Ambrevar** [comment]
This has been implemented in 1.2.1 (thanks to @FrozenPigs)!  You can now change the default search engines (duckduckgo and wikipedia).  To add new search engines:

```
(nconc (eval (get-default 'window 'search-engines))
       '(("q" . "http://quickdocs.org/search?q=~a")
         ("yt" . "https://www.youtube.com/results?search_query=~a")))
```

Later I plan to merge the search engine feature with bookmarks.  See #79 for more details.

**Comment by vindarel** [comment]
and to set the list:

```
(setf (get-default 'window 'search-engines)
      '(("default" . "https://duckduckgo.com/?q=~a")
        ("yt" . "https://www.youtube.com/results?search_query=~a")
        ("wiki" . "https://en.wikipedia.org/w/index.php?search=~a")))
```
from https://github.com/atlas-engineer/next/pull/198
